docs/source/reference/envvars.rst

```diff
@@ -430,6 +430,20 @@ GPU support
    Setting this variable to 1 also includes the values of arguments to Driver
    API calls in the logs.
 
+.. envvar:: NUMBA_NPY_RELAXED_STRIDES_CHECKING
+
+   By default Numba device arrays compute their contiguity using relaxed strides
```

Comment:
This will impact ROCm and anything else that inherits from `numba.misc.dummyarray.Array`? Perhaps:
```
By default arrays which inherit from `numba.misc.dummyarray.Array` (e.g. CUDA and ROCm device arrays) compute their...
```
?